packages/dependency-extraction-webpack-plugin/test/__snapshots__/build.js.snap

```diff
@@ -32,7 +32,7 @@ exports[`DependencyExtractionWebpackPlugin Webpack \`combine-assets\` should pro
 `;
 
 exports[`DependencyExtractionWebpackPlugin Webpack \`dynamic-import\` should produce expected output: Asset file 'main.asset.php' should match snapshot 1`] = `
-"<?php return array('dependencies' => array('wp-blob'), 'version' => 'c8be4fceac30d1d00ca7');
```

Comment:
I was curious why a Node upgrade would cause change of production JS assets. That shouldn't happen. And tracked it down to `caniuse-lite` upgrade:
- what actually changed in the bundle is that `globalThis.webpackChunk` references changed to `self.webpackChunk`.
- they changed because our supported browsers no longer all support `globalThis`.
- the browser that doesn's support `globalThis` is `and_uc` -- Android UC browser.
- we support this browser because it has more than 1% usage.
- it's only in the latest `caniuse-lite` data where this browser went over 1% -- it's 1.044%. It was under 1% in the old version.

A standalone PR that updates `caniuse-lite` (and maybe also pins it in `package.json`?) would be nice, but not necessary.

Reply:
I merged the `caniuse-lite`/`browserslist`/`core-js` upgrade separately in #52420. The newly supported browser (`and_uc`) caused different output for webpack runtime code (the hashes in test snapshots agree!), and also new CSS autoprefixes like `-webkit-margin-before` for `margin-block-start`.